Update the game over message (#1)
ALbum3270/skills-review-pull-requests: README.md

</header>

<!--
  <<< Author notes: Step 2 >>>

>   Start this step by acknowledging the previous step.

Author: ```suggestion
  Start this step by acknowledging the previous step.
  apple
```

  Define terms and link to docs.github.com.

>   Historic note: this step combines the commend, approve, and needs changes steps from the previous version.

Author: ```suggestion
  Historic note: this step combines the commend, approve, and needs changes steps from the previous version.
  hello1
```

-->

> ## Step 2: Assign yourself

Author: ```suggestion
## Step 2: Assign yourself
orange
```

> 

Author: hello world

_Great job opening that pull request! :wave:_

> **What is a _pull request review_?**: Reviewing a pull request is an opportunity to examine another contributor's changes and give them feedback. It's an awesome opportunity to learn more about how the project works and how others solve problems.

Author: ```suggestion= them feedback. It's an awesome opportunity to learn more about how the project works and how others solve problems.
```

> The best way to get a review is to ask for one. On GitHub, you can ask someone to review a pull request by assigning them as a reviewer or assignee. If you are not ready for review, consider [creating a draft pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request) instead.

Author: ```suggestion
The best way to get a review is to ask for one. On GitHub, you can ask someone to review a pull request by assigning them as a reviewer or assignee. If you are not ready for review, consider [creating a draft pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request) instead.
```
这是建议2

### :keyboard: Activity: Assign yourself

1. Open the pull request you just created.

> 1. Under **Assignees** on the right side of the screen, add yourself.

Author: ```suggestion
1. Under **Assignees** on the right side of the screen, r
```
```suggestion
1. Under **Assignees** on the right side of the screen
```
```suggestion
1. Under **Assignees** on the right side of the screen, add yourself.
```
```suggestion
1. Under **Assignees** on the right side of the screen, add yourself.
```
```suggestion
1. Under **Assignees** on the right side of the screen, add yourself.
```
```suggestion
1. Under **Assignees** on the right side of the screen, add yourself.
```
```suggestion
1. Under **Assignees** on the right side of the screen, add yourself.
```
```suggestion
1. Under **Assignees** on the right side of the screen, add yourself.
```
```suggestion
1. Under **Assignees** on the right side of the screen, add yourself.
```

In this course, you will:
The best way to get a review is to ask for one. On GitHub, you can ask someone to review a pull request by assigning them as a reviewer or assignee. If you are not ready for review, consider [creating a draft pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request) instead.

1. Open a pull request
2. Assign yourself
3. Leave a review
4. Suggest changes
5. Apply changes
6. Merge your pull request
### :keyboard: Activity: Assign yourself

### How to start this course
1. Open the pull request you just created.
1. Under **Assignees** on the right side of the screen, add yourself.

<!-- For start course, run in JavaScript:
'https://github.com/new?' + new URLSearchParams({
  template_owner: 'skills',
  template_name: 'review-pull-requests',
  owner: '@me',
  name: 'skills-review-pull-requests',
  description: 'My clone repository',
  visibility: 'public',

> }).toString()

Author: 这是评论1

-->

> [![start-course](https://user-images.githubusercontent.com/1221423/235727646-4a590299-ffe5-480d-8cd5-8194ea184546.svg)](https://github.com/new?template_owner=skills&template_name=review-pull-requests&owner=%40me&name=skills-review-pull-requests&description=My+clone+repository&visibility=public)

Author: 这是评论2

> 1. Wait about 20 seconds then refresh this page (the one you're following instructions from). [GitHub Actions](https://docs.github.com/en/actions) will automatically update to the next step.

Author: ```suggestion
1. Wait about 20 seconds then refresh this page (the one you're following instructions from). [GitHub Actions](https://docs.github.com/en/actions) will automatically update to the next step.
```
这是建议1

> 1. Wait about 20 seconds then refresh this page (the one you're following instructions from). [GitHub Actions](https://docs.github.com/en/actions) will automatically update to the next step.

Author: ```suggestion
1. Wly update to the next step.
```
